src/backend/dx12/src/command.rs

```diff
@@ -529,20 +529,28 @@ impl CommandBuffer {
         let cmd_buffer = &mut self.raw;
 
         // Bind vertex buffers
-        // We currently don't support offsets for vertex buffer binding, therefore,
-        // we only need to find out how many vertex buffer we need to bind.
-        let num_vbs = self.vertex_buffer_views
-            .iter()
-            .position(|view| view.SizeInBytes == 0)
-            .unwrap_or(MAX_VERTEX_BUFFERS);
-
-        unsafe {
-            cmd_buffer.IASetVertexBuffers(
-                0,
-                num_vbs as _,
-                self.vertex_buffer_views.as_ptr(),
-            );
+        let mut vertex_buffer_views = &self.vertex_buffer_views[..];
+        loop {
```

Comment:
Logic looks sane, could you document it a bit if possible? A few lines would be totally enough!

Reply:
(optionally: would be nice to avoid resetting all vertex buffers on each draw call. We didn't have it in the implementation so far.)